new-dual-repo.sh

```diff
@@ -288,24 +295,21 @@ git commit -m "Added bare README"	|| die "git commit failed"
 
 if [[ ! -e .gitignore ]]
 then
-	# create a sane .gitignore for Mac users
-
-cat <<EOINPUT > .gitignore
-.DS_Store
-.DS_Store?
-.Spotlight-V100
-.Trashes
-Icon
-Icon*
-EOINPUT
-
+	# create a .gitignore
+	if [[ -e "$ROOT_DIR/config/gitignore" ]]
+	then
+		gitignore=$(<"$ROOT_DIR/config/gitignore")
+		echo "$gitignore" > .gitignore
+	else
```

Comment:
I think it's more intuitive and less prone to mistakes to edit a template file than to edit heredocs.